pyproject.toml

```diff
@@ -12,6 +12,7 @@ python = ">=3.9,<3.13"
 python-dotenv = "^1.0.0"
 openai = "^1.2.4"
 graphviz = "^0.20.1"
+pydantic = "2.5.2"
```

Comment:
let's not lock a version, but use carrot case. This can be problematic if someone use another package with a different pydantic version.